src/client.ts

```diff
@@ -21,10 +21,20 @@ export class MongoClient {
     if (typeof options === "string") {
       options = parse(options);
     }
-    const conn = await Deno.connect({
+    var conn;
```

Comment:
Do not use `any` at will. Don't use `var`

Reply:
Ok. Fixed.